lib/rules/no-unused-class.js

```diff
@@ -107,14 +109,14 @@ export default {
            ```
            then the loop will be run 2 times
          */
-        _.forOwn(map, (o) => {
+        _.forIn(map, (o) => {
```

Comment:
Remember that all this functionality will run on each file save. We want it to be as performant as possible.